packages/api/src/utils.js

```diff
@@ -86,11 +88,49 @@ const decodeStateTransition = async (client, base64) => {
 
       break
     }
+    case StateTransitionEnum.MASTERNODE_VOTE: {
+      decoded.contestedResourcesVotePoll = stateTransition.getContestedDocumentResourceVotePoll().indexValues.map(buff => buff.toString('base64'))
+      decoded.contractId = stateTransition.getContestedDocumentResourceVotePoll().contractId.toString()
+      decoded.modifiedDataIds = stateTransition.getModifiedDataIds().map(identifier => identifier.toString())
+      decoded.ownerId = stateTransition.getOwnerId().toString()
+
+      break
+    }
     default:
       throw new Error('Unknown state transition')
   }
 
   return decoded
 }
 
-module.exports = { hash, decodeStateTransition, getKnex }
+const checkTcpConnect = (port, host) => {
+  return new Promise((resolve) => {
+    let connection
+    try {
+      connection = net.createConnection(port, host)
+
+      connection.setTimeout(TCP_CONNECT_TIMEOUT)
+
+      connection.once('error', async (e) => {
+        await connection.destroy()
+        resolve('ERR_CONNECTION_REFUSED')
```

Comment:
You should extract message from the error

Reply:
✅